package.json

```diff
@@ -39,7 +39,7 @@
     "update-versions": "lerna version --yes --no-commit-hooks --no-push --exact --conventional-commits --no-git-tag-version",
     "commit": "git-cz",
     "coverage": "codecov || exit 0",
-    "add-cli-no-save": "yarn add @aws-amplify/cli-internal@^11.0.0-zzz-beta.0 -W  && git checkout -- package.json",
+    "add-cli-no-save": "yarn add @aws-amplify/cli-internal@^11.0.0 -W  && git checkout -- package.json",
```

Comment:
Probably don't need the qualifier here, but I'm fine leaving it to make it explicit.

Reply:
Unless we rollback :D
Making it explicit at least revels this place if somebody is full text searching for versions.

Reply:
I removed it per @phani-srikar request.